demoapp/src/main/java/me/uport/sdk/demoapp/request_flows/PersonalSignRequestActivity.kt

```diff
@@ -0,0 +1,71 @@
+package me.uport.sdk.demoapp.request_flows
+
+
+import android.os.Bundle
+import android.support.v7.app.AppCompatActivity
+import android.view.View
+import com.uport.sdk.signer.KPSigner
+import kotlinx.android.synthetic.main.request_flow.*
+import kotlinx.coroutines.GlobalScope
+import kotlinx.coroutines.launch
+import kotlinx.coroutines.withContext
+import me.uport.sdk.core.UI
+import me.uport.sdk.demoapp.R
+import me.uport.sdk.jwt.JWTTools
+import me.uport.sdk.transport.Transports
+
+/**
+ *
+ * This activity demonstrates the flow for creating and sending a [Personal Signature Request]
+ *
+ **/
+
+class PersonalSignRequestActivity : AppCompatActivity() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.request_flow)
+
+        // instantiate signer
+        val signer = KPSigner("0x1234")
+
+        // create issuer DID
+        val issuerDID = "did:ethr:${signer.getAddress()}"
+
+        // create the request JWT payload
+        val payload = mapOf<String, Any>(
```

Comment:
good compromise showing how to build these requests even though we're missing the static types for personal sign